proof/crefine/RISCV64/IpcCancel_C.thy

```diff
@@ -2948,198 +3035,256 @@ lemma cancelIPC_ccorres1:
                    (UNIV \<inter> {s. tptr_' s = tcb_ptr_to_ctcb_ptr thread}) []
           (cancelIPC thread) (Call cancelIPC_'proc)"
   apply (cinit lift: tptr_' simp: Let_def cong: call_ignore_cong)
-sorry (* FIXME RT: cancelIPC_ccorres1 *) (*
    apply (rule ccorres_move_c_guard_tcb)
-   apply csymbr
+   apply (rule ccorres_stateAssert)
+   apply (rule ccorres_stateAssert)
```

Comment:
trivial: can say `apply (rule ccorres_stateAssert)+` as a message of "we're going through all the asserts here"

Reply:
Done.